src/main/java/org/mskcc/kickoff/printer/PatientFilePrinter.java

```diff
@@ -57,4 +62,46 @@ protected String getOutputFilenameEnding() {
     protected Predicate<Sample> getSamplePredicate() {
         return sample -> true;
     }
+
+    @Override
+    protected void writeExternalTumors(KickoffRequest kickoffRequest, StringBuilder outputText) {
+        for (KickoffExternalSample externalSample : kickoffRequest.getTumorExternalSamples()) {
+            //pool
+            outputText.append("NA");
+            outputText.append("\t");
+
+            outputText.append(Utils.sampleNormalization(externalSample.getCmoId()));
+            outputText.append("\t");
+
+            outputText.append(externalSample.getExternalId().replace('-', '_'));
+            outputText.append("\t");
+
+            outputText.append(Utils.patientNormalization(externalSample.getPatientCmoId()));
+            outputText.append("\t");
+
+            outputText.append(getIfAvailable(externalSample.getSampleClass()));
+            outputText.append("\t");
+
+            outputText.append(getIfAvailable(externalSample.getSpecimenType()));
+            outputText.append("\t");
+
+            //input_ng
+            outputText.append("NA");
```

Comment:
input_ng library_yield pool_input , in patient file will always NA?

Reply:
yep, we don;t have those values and Oliver said it's better to leave "NA"

Reply:
will this PR generate correct patient file, if yes, i will merge it...